debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/ShowDumpAgentsCommand.java

```diff
@@ -146,12 +150,12 @@ public ShowDumpAgentsCommand()
 //		{ "traceassert", "ON_TRACE_ASSERT",         J9RAS_DUMP_ON_TRACE_ASSERT },
 		rasDumpEventsToNames.put(J9RAS_DUMP_ON_TRACE_ASSERT, "traceassert");
 //		{ "allocation",  "ON_OBJECT_ALLOCATION",    J9RAS_DUMP_ON_OBJECT_ALLOCATION },
-		rasDumpEventsToNames.put(J9RAS_DUMP_ON_OBJECT_ALLOCATION,  "allocation");
-//		{ "corruptcache",  "ON_CORRUPT_CACHE",   	J9RAS_DUMP_ON_CORRUPT_CACHE }
+		rasDumpEventsToNames.put(J9RAS_DUMP_ON_OBJECT_ALLOCATION, "allocation");
+//		{ "corruptcache",  "ON_CORRUPT_CACHE",   	  J9RAS_DUMP_ON_CORRUPT_CACHE }
 		rasDumpEventsToNames.put(J9RAS_DUMP_ON_CORRUPT_CACHE, "corruptcache");
 
 		/* Known VM requests */
-//		{ "multiple",  "DO_MULTIPLE_HEAPS",        J9RAS_DUMP_DO_MULTIPLE_HEAPS },
+//		{ "multiple",  "DO_MULTIPLE_HEAPS",          J9RAS_DUMP_DO_MULTIPLE_HEAPS },
```

Comment:
If you're going to adjust spacing so things line up, please remove all internal tabs (those other than used for indentation).